dev-docs/show-outstream-video-ads.md

```diff
@@ -115,6 +116,36 @@ pbjs.addAdUnit({
 
 {% endhighlight %}
 
+In a multiFormat adUnit, you might want the renderer to only apply to only one of the mediaTypes.  You can do this by defining the renderer on the media type itself.
+{% highlight js %}
+
+pbjs.addAdUnit({
+    code: 'video1',
+    // This renderer would work the same as it does above...
+    renderer: {
+        url: 'example.com/publishersCustomRenderer.js',
```

Comment:
i think we should make this an example of the new "preferred" way and remove this altogether? what do you think

Reply:
also I would change line 171 to refer to your pull request, 5760